src/moin/scripts/migration/moin19/_utils19.py

```diff
@@ -126,15 +126,11 @@ def unquoteWikiname(filename, charset=CHARSET):
     :rtype: unicode
     :returns: WikiName
     """
-    # From some places we get called with Unicode strings
-    if isinstance(filename, str):
-        filename = filename.encode(CHARSET)
-
     parts = []
     start = 0
     for needle in QUOTED.finditer(filename):
         # append leading unquoted stuff
-        parts.append(filename[start:needle.start()])
+        parts.append(bytes(filename[start:needle.start()].encode(charset)))
```

Comment:
this looks suspicious.

type of filename? guess this should be str, otherwise .encode() would not make sense.

after .encode(), we have bytes, so why another bytes() around that?

Reply:
will remove redundant bytes()